### PR TITLE
Warning message

### PR DESCRIPTION
**Pull request by tt-le**
Added warning message thrown by test case and explaining that it is expected.

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -413,13 +413,18 @@ Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 5.92 s - in it.i
...
+Could not send Message.
+[err] The specified host is unknown.
 Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.077 s - in it.io.openliberty.guides.inventory.InventoryEndpointIT
 
 Results:
 
 Tests run: 7, Failures: 0, Errors: 0, Skipped: 0
 ----
 
+The warning message is expected and results from a request to a bad or an unknown host name.
```
Comment (AustinSeto) [inline README.adoc:426]: From reviews of other similar changes, minor change to say "The warning messages and errors are expected" and to mention which test causes them.

**Review by AustinSeto — CHANGES_REQUESTED** [review]

**Review by AustinSeto — APPROVED** [review]
LGTM

**Review by gkwan-ibm — CHANGES_REQUESTED** [review]
@tt-le no need to hotspot the `testUnknownHost()` and `InventoryEndpointIT`. Backtick should be enough, because no need to highlight them.

**Review by griffinhadfield — APPROVED** [review]
Checked it out in lgdev. Looks good to me.

**Review by gkwan-ibm — APPROVED** [review]

**Pull request merged**